.github/workflows/ci-libshortfin.yml

```
@@ -64,6 +64,12 @@ jobs:
             python-version: "3.12"
             cmake-options:
               -DCMAKE_C_COMPILER=gcc-14 -DCMAKE_CXX_COMPILER=g++-14
+          - name: Ubuntu (GCC 14, tracing)
+            runs-on: ubuntu-24.04
+            # Only test with GCC 14 and Python 3.12
```

As you disabled testing (which is fine) we could rephrase to
```suggestion
            # Only build with GCC 14 and Python 3.12
```
But no strong need.